bigquery/indexer.py

```diff
@@ -117,6 +114,16 @@ def _docs_by_id(df, table_name, participant_id_col):
         yield row[participant_id_col], row_dict
 
 
+def _field_docs_by_id(table_name, fields):
+    for field in fields:
+        field_dict = {
+            "elasticsearch_field_name" : table_name + '.' + field.name,
+            "name" : field.name,
+            "description":field.description
+        }
+        yield field.name, field_dict
```

Comment:
After further thought, document id shouldn't be field.name, because that's not unique.

Let's have document id be elasticsearch_field_name, and remove elasticsearch_field_name from dict. (It's not needed in dict since it can be retrieved from document id.)